modules/nf-core/sentieon/bwamem/main.nf

```diff
@@ -47,6 +47,9 @@ process SENTIEON_BWAMEM {
         echo "Decoded and exported Sentieon test-license system environment variables"
     fi
 
+    LD_LIBRARY_PATH=/usr/local/lib/:\$LD_LIBRARY_PATH
```

Comment:
Maybe I should mention that if this is with conda, the path may be different. Not sure if the path works correctly with conda by default and that can be ignored.

Reply:
I think the default conda/docker works from what I can say.

Maybe let's try to do that only when using singularity/apptainer.

Reply:
Ok, I'll try to do it that way. Testing right now.